Adds new function to create an R-object (dataframe) with details of atlas definitions

gowthamrao opened the pull request:
Addresses https://github.com/OHDSI/ROhdsiWebApi/issues/40

schuemie commented on R/WebApi.R:
```diff
@@ -142,3 +142,99 @@ getCdmSources <- function(baseUrl) {
+#' returned as text/character (to be worked on in future version).
+#'
+#' @export
+getAtlasDefinitionDetails <- function(baseUrl) {
```
I would think 'getAtlasDefinitionsDetails`? You're very likely getting details for more than one definition.

schuemie commented:
Looks good except I'd change the function name to plural

gowthamrao commented:
Thanks. Updated. 

To do: get created-date, updated-date, modified-date in date-time format consistently. Right now it is being returned inconsistently by webapi - some are date-time objects, others are strings. Makes it difficult to systematically parse it

https://github.com/OHDSI/ROhdsiWebApi/blob/e97f2582fc7aa20d47890763d2b477f1cfc39b1f/R/WebApi.R#L224
